soroban-sdk/src/tests/budget.rs

```diff
@@ -27,6 +27,6 @@ fn test_budget() {
     // 2 - charge for adding the two elements
     // 1 - charge for binary search of map with len == 0
     // 2 - charge for binary search of map with len == 1
-    assert_eq!(e.budget().tracker(ContractCostType::MapEntry), (5, None));
+    assert_eq!(e.budget().tracker(ContractCostType::MapEntry), (8, None));
```

Comment:
@jayz22 is this expected?

Reply:
I believe that's due to the recent map changes in env. so yes.